cmake/FindCUDA/FindCUDA.cmake

```diff
@@ -1672,9 +1760,10 @@ function(CUDA_LINK_SEPARABLE_COMPILATION_OBJECTS output_file cuda_target options
       add_custom_command(
         OUTPUT ${output_file}
         DEPENDS ${object_files}
-        COMMAND ${CUDA_NVCC_EXECUTABLE} ${nvcc_flags} -dlink ${object_files} -o ${output_file}
+        COMMAND ${CUDA_NVCC_EXECUTABLE} ${nvcc_flags} -dlink ${object_files} ${CUDA_cublas_device_LIBRARY} -o ${output_file}
```

Comment:
this doesn't look right. Why is everything linking against `${CUDA_cublas_device_LIBRARY}` regardless. Either ways prob harmless change.

Reply:
This commit comes from:

```
commit 3726ded116e405d28d7d2bbd0abc1b929ab0adb0
Author: Jamil Appa <jamil.appa@zenotech.com>
Date:   Fri Nov 3 18:08:50 2017 -0400

    FindCUDA: Add cublas device library to separable compilation
    
    Add cublas libraries to separable compilation device link line to
    resolve missing symbols when using cublas device library.
```

https://gitlab.kitware.com/cmake/cmake/commit/3726ded116e405d28d7d2bbd0abc1b929ab0adb0

I'm not sure why they needed to special case this upstream, but I agree, it looks harmless.